javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,11 @@
+/* exported filterOutStrings */
+function filterOutStrings(values) {
+  var arr = [];
+
+  for (var i = 0; i < values.length; i++) {
+    if (typeof values[i] === 'number') {
```

Comment:
This isn't checking if the value isn't a string, it's checking if it is a `number`. What if I gave you other data types such as booleans or arrays?